src/Microsoft.Graph.Core/Requests/Content/BatchRequestContentSteps.cs

```diff
@@ -0,0 +1,253 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Graph;
+
+/// <summary>
+/// Represents a collection of ordered <see cref="BatchRequestStep"/> objects.
+/// </summary>
+public class BatchRequestContentSteps : IReadOnlyDictionary<string, BatchRequestStep>, IDictionary<string, BatchRequestStep>
```

Comment:
We just want to use this internally for now. 

```suggestion
internal class BatchRequestContentSteps : IReadOnlyDictionary<string, BatchRequestStep>, IDictionary<string, BatchRequestStep>
```